backend/src/main/resources/application.yaml

```diff
@@ -63,7 +63,9 @@ simple-report:
   sendgrid:
     enabled: false
     from-email: admin@simplereport.gov
```

Comment:
Should we change this to support@simplereport.gov since we've already verified that one?

Reply:
Yes, yes we should. Good catch!